addons/medical_gui/script_component.hpp

```diff
@@ -34,6 +34,8 @@
 #define PATIENT_INFO_IGUI_OFFSET_X (PATIENT_INFO_IGUI_X - PATIENT_INFO_IGUI_BASE_X)
 #define PATIENT_INFO_IGUI_OFFSET_Y (PATIENT_INFO_IGUI_Y - PATIENT_INFO_IGUI_BASE_Y)
 
+#define PAGE_COUNT ((count GVAR(possibleActions))*9)
```

Comment:
I feel like this should be division? (not sure)

Reply:
yep, my bad